mathics/builtin/assignments/internals.py

```diff
@@ -530,19 +530,24 @@ def process_assign_format(self, lhs, rhs, evaluation, tags, upset):
         evaluation.message_args("Format", len(lhs.elements), 1, 2)
         raise AssignmentException(lhs, None)
     if len(lhs.elements) == 2:
-        form = lhs.elements[1].get_name()
-        if not form:
+        form = lhs.elements[1]
+        form_names = form.get_name()
```

Comment:
Why not just `form_name`?

Reply:
@TiagoCavalcante, I put it in plural because some lines below a list is assigned to the same variable (in the case that the Form is not specified).